modAionImpl/src/org/aion/equihash/OptimizedEquiValidator.java

```diff
@@ -58,23 +51,23 @@ public OptimizedEquiValidator(int n, int k) {
         this.indicesHashLength = (n + 7) / 8;
         this.hashOutput = indicesPerHashOutput * indicesHashLength;
         this.collisionBitLength = n / (k + 1);
-        this.collisionByteLength = (collisionBitLength + 7) / 8;
+        // int collisionByteLength = (collisionBitLength + 7) / 8;
```

Comment:
were these commented-out pieces meant to be deleted?

Reply:
IntelliJ's code inspection pointed out that they were not used. They may become useful in the future, which is why I only commented the code.